homework-submission/myapp/src/Components/Week3/Components/Index.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import List from './List';
+
+class HomeworkWeek3 extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      items: [],
+    };
+  }
+
+  componentDidMount() {
+    const handleErrors = response => {
+      if (!response.ok) {
+        throw Error(response.statusText);
+      }
+      return response;
+    };
+
+    fetch('https://uinames.com/api/?amount=10')
+      .then(handleErrors)
+      .then(res => res.json())
+      .then(items =>
+        this.setState(
+          {
+            error: handleErrors,
+            isLoaded: true,
+            items,
+          },
+          () => console.log(items),
+        ),
+      )
+      .catch(error => this.setState({ error, isLoading: false }));
```

Comment:
Also you have to change error to true or a message so we know something is went wrong

Also in render make a condition if error !== null then return error message